src/fragments/lib-v1/storage/js/list.mdx

```diff
@@ -96,18 +96,44 @@ Storage.list('photos/', { level: 'private' })
     .catch(err => console.log(err));
 ```
 
-## Access all files 
+## Access all files
 
-To get a list of all files in your S3 bucket under a specific prefix, you can set maxKeys to ALL. The default value of maxKeys is 1000.
+To get a list of all files in your S3 bucket under a specific prefix, you can set pageSize to ALL. The default value of pageSize is 1000.
 
 ```javascript
 
-Storage.list('photos/', { maxKeys : 'ALL' })
-.then(result => console.log(result))
-.catch(err => console.log(err));;
+Storage.list('photos/', { pageSize : 'ALL' })
+  .then(({ results }) => console.log(result))
+  .catch((err) => console.log(err));
+
+```
+
+## Paginated file access
+
+If the the pageSize is set to lower than total file size, a single `Storage.list` call only returns a subset of all the files. To list all the files with multiple calls, users can use the `hasNextToken` and `nextToken` key:
```

Comment:
`the the` -> `the`
`set to lower` -> `set lower`
`than total file size` -> `than the total file size`
`key` -> `keys`